pkg/scheduler/framework/statement.go

```diff
@@ -225,20 +248,21 @@ func (s *Statement) UnPipeline(task *api.TaskInfo) error {
 
 // Allocate the task to node
 func (s *Statement) Allocate(task *api.TaskInfo, nodeInfo *api.NodeInfo) (err error) {
+	errInfos := make([]error, 0)
+	hostname := nodeInfo.Name
+
 	podVolumes, err := s.ssn.cache.GetPodVolumes(task, nodeInfo.Node)
 	if err != nil {
-		return err
+		klog.Errorf("Failed to get pod volume for task %v/%v on node %v when allocating in Session <%v>: %v",
+			task.Namespace, task.Name, hostname, s.ssn.UID, err)
+		errInfos = append(errInfos, err)
 	}
 
-	hostname := nodeInfo.Name
 	if err := s.ssn.cache.AllocateVolumes(task, hostname, podVolumes); err != nil {
-		return err
+		klog.Errorf("Failed to allocate volume for task %v/%v on node %v when allocating in Session <%v>: %v",
+			task.Namespace, task.Name, hostname, s.ssn.UID, err)
+		errInfos = append(errInfos, err)
 	}
-	defer func() {
```

Comment:
Can this be deleted directly? Please make sure these rollback  can be done by framework. @JesseStutler

Reply:
Yes, framework does in `unallocate`:
https://github.com/volcano-sh/volcano/blob/549ad9ac5aff96343bc5067fa9f736c257d5746b/pkg/scheduler/framework/statement.go#L316-L317